docs/fa/README-FA.md

```diff
@@ -0,0 +1,105 @@
+# (یادگیری عمیق(با پایتورچ [![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/Atcold/pytorch-Deep-Learning/master)
+&#x202b;
+این مخزن یک [وبسایت](https://atcold.github.io/pytorch-Deep-Learning/) به همراه خود دارد که تمام محتویات دوره در قالب متن و ویدیو در دسترس است.
+
+<!-- Mandarin -->
+<!--
+🇨🇳 这份`README.md`的中文版本及网站可以在[这里](https://github.com/Atcold/pytorch-Deep-Learning/blob/master/docs/zh/README-ZH.md)找到.-->
+
+<!-- Korean -->
+<!--
+🇰🇷 한국어 `README.md`는 [여기](https://github.com/Atcold/pytorch-Deep-Learning/blob/master/docs/ko/README-KO.md)에서 확인하실 수 있습니다.
+-->
+
+<!-- Spanish -->
+<!--
+🇪🇸 La versión en Español de este `README.md` y el sitio web [se encuentran aquí](https://github.com/Atcold/pytorch-Deep-Learning/blob/master/docs/es/README-ES.md).-->
+
+<!-- Italian -->
+<!--
+🇮🇹 La versione in italiano di questo `README.md` può essere trovata  [a questo indirizzo](https://github.com/Atcold/pytorch-Deep-Learning/blob/master/docs/it/README-IT.md).
+-->
+<!-- Turkish -->
+<!--
+🇹🇷 `README.md`'nin Türkçe çevirisi [bu](https://github.com/Atcold/pytorch-Deep-Learning/blob/master/docs/tr/README-TR.md) adreste bulunabilir.
+-->
+<!-- Japanese -->
+<!--
+🇯🇵 日本語版の `README.md` は  [ここ](https://github.com/Atcold/pytorch-Deep-Learning/blob/master/docs/ja/README-JA.md) にあります.
+-->
+
+<!-- Arabic -->
+<!--
+🇸🇦 النسخة العربية من ملف `README.md`  [ar](https://github.com/Atcold/pytorch-Deep-Learning/blob/master/docs/ar/README-AR.md) والموقع الالكتروني.
+-->
+
+<!-- Farsi -->
+<!--
+🇫🇦 نسخه فارسی `README.md`  [ar](https://github.com/Atcold/pytorch-Deep-Learning/blob/master/docs/ar/README-AR.md) در وبسایت.
+-->
+
+
+<!-- English - Mandarin - Korean - Spanish - Italian - Turkish - Japanese - Arabic - Farsi -->
+[🇬🇧](https://github.com/Atcold/pytorch-Deep-Learning/blob/master/README.md) &nbsp; [🇨🇳](https://github.com/Atcold/pytorch-Deep-Learning/blob/master/docs/zh/README-ZH.md) &nbsp; [🇰🇷](https://github.com/Atcold/pytorch-Deep-Learning/blob/master/docs/ko/README-KO.md) &nbsp; [🇪🇸](https://github.com/Atcold/pytorch-Deep-Learning/blob/master/docs/es/README-ES.md) &nbsp; [🇮🇹](https://github.com/Atcold/pytorch-Deep-Learning/blob/master/docs/it/README-IT.md) &nbsp; [🇹🇷](https://github.com/Atcold/pytorch-Deep-Learning/blob/master/docs/tr/README-TR.md) &nbsp; [🇯🇵](https://github.com/Atcold/pytorch-Deep-Learning/blob/master/docs/ja/README-JA.md) &nbsp; [🇸🇦](https://github.com/Atcold/pytorch-Deep-Learning/blob/master/docs/ar/README-AR.md)&nbsp; [🇫🇦](https://github.com/Atcold/pytorch-Deep-Learning/blob/master/docs/fa/README-FA.md)
```

Comment:
Please, add :fr: and let's use :iran: for Persian?

Reply:
OK, added 🇫🇷 , and used 🇮🇷 instead.